app/controllers/flyover_comments/comments_controller.rb

```diff
@@ -63,6 +71,10 @@ def load_commentable
       end
     end
 
+    def authorize_flyover_comment_show!
+      raise "User isn't allowed to view comment" unless can_view_flyover_comment?(@comment, send(FlyoverComments.current_user_method.to_sym))
```

Comment:
What's the terminology we want to have for this, since the show action only returns the replies. And do we want some sort of param for rendering top level as well?